.gitignore

```diff
@@ -31,6 +31,7 @@ presto-docs-venv/
 # presto-native-execution
 #==============================================================================#
 _build/
+cmake-build-*/
```

Comment:
Can we make it more specific, i.e.

```
cmake-build-debug
cmake-build-release
```

?

Reply:
I think it's hard for us to predict here. A new entry in this Preference page would have a new cmake-build-* directory? https://www.internalfb.com/intern/px/p/2c6p9